### ISSUE TITLE
These Mods Might not interact well together...

### ISSUE THREAD

**Issue by MC-Nerd** [ldtteam/Domum-Ornamentum#270]
So I'm playing this Christmas mod with my friend, and I accidentally shifted + clicked on a trapdoor (made with the architect's cutter from this mod) and it bricked my game. Unplayable now as I cannot put down the trapdoor and I physically cannot click, scroll through inventory or anything. My friend and I tried a few different things to see if we could save my game; disabling the mods, doing the tp hack, even having my friend kill me and see if it respawns me without the door (terrible idea, you will be stuck on the respawn screen forever) but to no avail. I tested it in a new world to see if it would happen again (pictured below) and sure enough, I crafted an Acacia log trapdoor and shift-clicked, stuck. If anyone knows a solution, please let me know, but otherwise heed my warning, this mod won't do well with the 'Carry On' Mod as these seem to behave like entities and not blocks. I was trying to upload a short 12 sec video as well, but the file size is still too large, apologies. In the video the object (trapdoor) in my hand seems to change colors at short intervals, definitely a major indication that something has gone wrong.

R.I.P Christmas World 2024 :(

![Screenshot 2024-12-24 174213](https://github.com/user-attachments/assets/2a684c03-334f-4341-8d75-e6536c6a5cd3)

**Comment by Talyda** [comment]
Yeah, nah, never pick up a DO block with carryon.
I have a very vague memory of a carryon command? /carryon clear maybe? Does that work?

**Comment by Talyda** [comment]
Yeah, that was command I saw mentioned. Someone else started the game with carryon disabled, and that worked for them

**Comment by MC-Nerd** [comment]
> Yeah, that was command I saw mentioned. Someone else started the game with carryon disabled, and that worked for them

Gonna try these, thank you so much! I had so much progress in that world I legit almost cried when it happened lol. Brb.

**Comment by Talyda** [comment]
Oh, I totally understand that! Hopefully one of them works!

**Comment by MC-Nerd** [comment]
> Oh, I totally understand that! Hopefully one of them works!

YOU ARE A LIFESAVERRR!!! Thank you kindly <3. Hope you have a great holiday and Happy New Year.

**Comment by Talyda** [comment]
Oh, I am so glad it worked! Happy holidays to you too!
(which one worked BTW? Jut in case anyone comes here looking in the future!)

**Comment by Nightenom** [comment]
Could you please send us the log when it was happening? If u don't know which one it was then it should be easy to replicate in a new world. Thank you

**Comment by vampyress** [comment]
> Could you please send us the log when it was happening? If u don't know which one it was then it should be easy to replicate in a new world. Thank you

https://gist.github.com/vampyress/84621e2dc83c4b6d01909d9447010ad5

**Comment by Talyda** [comment]
Vampyress had same problem Night, so I asked them to pop their log in here for you
